src/components/ColorCard.svelte

```diff
@@ -33,22 +38,38 @@
     colorName = nearest(color.toString('hex')).name
   }
 
-  const onColorInput = async (event) => {
-    colorInput = event.target.value
-    color = new Color(colorInput)
-    inputFormat = new TinyColor(colorInput).format
+  /*
+    Check if passed argument is a valid color and update `isValidColor` accordingly
+  */
+  const validateColor = (value: string) => isValidColor = chroma.valid(value)
+
+  /*
+    Set passed color as active if it's a valid color
+  */
+  const setNewColor = async (newColor: string) => {
```

Comment:
I'm not so sure about the name of the function, maybe `updateColor` would fit a bit better. What do you think?

Reply:
Yeah, will fix that